File: app/services/omni/sku_cost.rb
class Omni::SkuCost::Service
  include Buildit::Service::Base

  service 'SkuCost', 'Omni.service'

  connected_mode(Buildit::Service::Backend::Crud) do |config|
    config.model = Omni::SkuCost
  end

end # class Omni::SkuCost::Service

File: app/assets/javascripts/omni_desktop/store/SkuCost.js
Ext.define('Omni.store.SkuCost', {
  extend       : 'Ext.data.Store',
  alias        : 'store.omni-SkuCost',
  model        : 'Omni.model.SkuCost',
  autoLoad     : false,
  storeId      : 'SkuCostStore',
  remoteFilter : true,
  remoteSort   : true,

  constructor  : function (config) {
    var me = this;
    me.callParent(this);
    var proxy = Ext.Object.merge({}, me.getProxy());
    me.setProxy(proxy);
  } // constructor

}); // Ext.define